keras/distribution/distribution_lib.py

```diff
@@ -130,7 +130,7 @@ class TensorLayout:
     """
 
     def __init__(self, axes, device_mesh=None):
-        self._axes = axes
+        self._axes = list(axes)
```

Comment:
Likewise we want to standardize on a tuple here (unless we ever have to edit `axes` dynamically)

Reply:
Done.